Mathlib/Data/Pi/Algebra.lean

```diff
@@ -187,7 +187,7 @@ variable [DecidableEq I]
 variable [∀ i, One (f i)] [∀ i, One (g i)] [∀ i, One (h i)]
 
 /-- The function supported at `i`, with value `x` there, and `1` elsewhere. -/
-@[to_additive Pi.Single "The function supported at `i`, with value `x` there, and `0` elsewhere."]
+@[to_additive "The function supported at `i`, with value `x` there, and `0` elsewhere."]
```

Comment:
Is there a reason for not doing
```
#align pi.mul_single Pi.mulSingle
#align pi.single Pi.single
```
after this declaration?

Reply:
Not as far as I know, good catch! (When I modified this file I only added additive aligns to definitions with existing aligns; I guess this one didn't an original one.) I've added one now.